tests/test_alloydb_vectorstore_index.py

```diff
@@ -123,13 +167,41 @@ async def test_dropindex(self, vs):
         assert not result
 
     async def test_aapply_vector_index_ivfflat(self, vs):
-        index = IVFFlatIndex(distance_strategy=DistanceStrategy.EUCLIDEAN)
+        index = IVFFlatIndex(distance_strategy=DistanceStrategy.PGVECTOR_EUCLIDEAN)
         await vs.aapply_vector_index(index, concurrently=True)
         assert await vs.is_valid_index(DEFAULT_INDEX_NAME)
         index = IVFFlatIndex(
             name="secondindex",
-            distance_strategy=DistanceStrategy.INNER_PRODUCT,
+            distance_strategy=DistanceStrategy.PGVECTOR_INNER_PRODUCT,
         )
         await vs.aapply_vector_index(index)
         assert await vs.is_valid_index("secondindex")
         await vs.adrop_vector_index("secondindex")
+        await vs.adrop_vector_index()
+
+    async def test_aapply_vector_index_ivf(self, vs):
+        index = IVFIndex(distance_strategy=DistanceStrategy.PGVECTOR_EUCLIDEAN)
+        await vs.aapply_vector_index(index, concurrently=True)
+        assert await vs.is_valid_index(DEFAULT_INDEX_NAME)
+        index = IVFIndex(
+            name="secondindex",
+            distance_strategy=DistanceStrategy.PGVECTOR_INNER_PRODUCT,
+        )
+        await vs.aapply_vector_index(index)
+        assert await vs.is_valid_index("secondindex")
+        await vs.adrop_vector_index("secondindex")
+        await vs.adrop_vector_index()
```

Comment:
We shouldn't need both drop methods

Reply:
If we do not drop the first index, it raises `asyncpg.exceptions.DuplicateTableError: relation "langchainvectorindex" already exists` error because the default index name is the same. We can specify index name when initializing the first index, but then we don't test for initializing without index name.

Reply:
Thanks for explanation. Please disregard comment.